src/components/Spreader/Spreader.tsx

```diff
@@ -0,0 +1,33 @@
+import { useStyles } from '@helpers/hooks/useStyles'
+import React, { FC } from 'react'
+
+import styles from './Spreader.module.scss'
+
+interface SpreaderProps {
+  className?: string | string[]
+  spread?:
+    | 'mini'
+    | 'tiny'
+    | 'small'
+    | 'medium'
+    | 'large'
+    | 'x-large'
+    | 'big'
+    | 'huge'
+}
+
+const Spreader: FC<SpreaderProps> = ({ className, spread = 'medium' }) => {
```

Comment:
w pozostałych plikach z tego co widziałam, jest koncepcja, że domyślne className to ''